src/tls.c

```diff
@@ -3123,6 +3124,21 @@ static int TLSX_CSR_Parse(WOLFSSL* ssl, byte* input, word16 length,
 
     #if defined(WOLFSSL_TLS13) && !defined(NO_WOLFSSL_SERVER)
         if (ssl->options.tls1_3) {
+            cert = (DecodedCert*)XMALLOC(sizeof(DecodedCert), ssl->heap, DYNAMIC_TYPE_DCERT);
```

Comment:
Not sure if what I've done here is the most efficient way to do it. The server is decoding its own certificate here and ultimately using the decoded data to populate an OcspRequest (including the URL of the OCSP responder). Does the decoded cert already exist on the server side somewhere?